omod/src/main/webapp/pages/myaccount/changeDefaults.gsp

```diff
@@ -8,10 +8,40 @@
         primaryLocaleOptions.add([label: it.displayName, value: it.toString()])
     }
 %>
+<script type="text/javascript">
 
+  var value;
+  
+  jq(document).ready(function() {
+    jq("select").on('change', function() {
+	    value = this.value;
+	    if (value.indexOf("_") >- 1) {
+	        value = value.slice(0, value.indexOf("_"));
+	    }
+        getCheckBoxValues();
+    });
+  });
+  
+  function getCheckBoxValues() {
+     jq("[name='proficientLocales']").each(function() { 
+		 if (this.value == value) {
+		     unCheckInitialValue();
+		     jq(this).prop('checked',true);
+	     }
+	 });
+                              }
+  function unCheckInitialValue() {
+     jq("[name='proficientLocales']").each(function() {
+ 	     if (jq(this).prop('checked') == true) {
+ 	         jq(this).prop('checked',false);
+         }
+     });
+  }
+  
+</script>
 <script type="text/javascript">
 
-    var breadcrumbs = [
+   a var breadcrumbs = [
```

Comment:
What is the above "a" character for? Did you really test this and got no javascript errors?

Reply:
@dkayiwa  Oooohh sorry, this error came up during indention